test/functional/legacy/delete_spec.lua

```diff
@@ -44,6 +42,10 @@ describe('Test for delete()', function()
   end)
 
   it('symlink delete', function()
+    if helpers.os_name() == 'windows' then
+      pending('No symlinks in Windows')
+      return
+    end
```

Comment:
Windows has `mklink`, so we can eventually update these tests. So let's use

    if helpers.pending_win32(pending) then return end

here (and below) so we remember to update these tests later.